test/unittest/tests/utils_extlist.cpp

```diff
@@ -247,3 +246,69 @@ verify_off_extent_list(
     REQUIRE(extlist.bytes == expected_bytes);
 }
 } // namespace utils.
+
+/*!
+ * operator<<(std::ostream &out, const utils::off_size *os) --
+ *     Print function for const utils::off_size *.
+ *
+ * @param out ostream
+ * @param os To be printed
+ */
+std::ostream &
+operator<<(std::ostream &out, const utils::off_size *os)
+{
+    if (os == nullptr) {
+        out << "(nullptr)";
+        return out;
+    }
+    out << "{off " << os->off << ", size " << os->size << ", end " << os->end() << '}';
+    return out;
+}
+
+/*!
+ * operator<<(std::ostream &out, const WT_EXT *ext) --
+ *     Print function for const WT_EXT *.
+ *
+ * @param out ostream
+ * @param ext To be printed
+ */
+std::ostream &
+operator<<(std::ostream &out, const WT_EXT *ext)
```

Comment:
It would be nice to create an WT_EXT and WT_EXTLIST class wrapper or place it into the struct class.

Reply:
What would an WT_EXT or WT_EXTLIST wrapper be for? They exist already in test/unittest/tests/test_extent_list.cpp.  They are only for managing memory allocation and deletion. However, they are incompatible with __ wti_block_ext_free and __ wti_block_ext_alloc. These __ wti functions cannot be avoided in many extent list functions since they are used internal to WiredTiger. I think those two wrappers should be deleted and replaced with ext_free_list.